src/utils/__tests__/fixtures/incidents.json

```diff
@@ -58,23 +60,43 @@
       "sub_slug": "wegsleep",
       "main": "Overlast in de openbare ruimte",
       "main_slug": "overlast-in-de-openbare-ruimte",
-      "department": "ASC, CCA, THO",
-      "category_url": "https://category-url.amsterdam.nl"
+      "departments": "ASC, CCA, THO",
+      "category_url": "https://category-url.amsterdam.nl",
+      "created_by": "another_user@domain.com",
+      "deadline": "2021-09-14T12:05:04.002163+02:00",
+      "deadline_factor_3": "2021-09-22T12:05:04.002163+02:00",
+      "text": null
     },
     "incident_date_start": "2018-12-03T10:41:42+01:00",
     "text_extra": "",
-    "assigned_user_email": "user1@example.com"
+    "assigned_user_email": "user1@example.com",
+    "has_attachments": false,
+    "has_children": false,
+    "has_parent": false,
+    "notes": [
+      {
+        "created_by": null,
+        "text": "Automatische e-mail bij registratie van de melding is verzonden aan de melder."
+      }
```

Comment:
I could be mistaken, but I'm not seeing the `notes` prop in the endpoint's output: https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml#/default/get_signals_v1_private_signals_.

Reply:
Good point - I'll ask to have this property added to the docs.

Reply:
This property will be added to the Swagger docs